src/components/Home/ScrollToHash.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react'
+import { useLocation } from 'react-router-dom'
+
+function ScrollToHash() {
```

Comment:
Why isn't this a hook `useScrollToHash()`?

Reply:
I can make it a hook instead

```typescript

function LayoutShell() {
  useScrollToHash()

  return (
      <IntlProvider defaultLocale="en" locale="en" messages={flattenMessages(en)}>
        <SnapshotStatus />
        <Layout>
          <Outlet />
        </Layout>
      </IntlProvider>
  )
}
```

Are there any pro/cons?

Reply:
No, but it doesn't look like a component :P